spec/models/order_cycle_spec.rb

```diff
@@ -525,15 +525,13 @@
     let!(:order5) { create(:completed_order_with_totals, distributor: shop, user: user, order_cycle: oc)  }
 
     before do
-      Spree::MailMethod.create!(
-        environment: Rails.env,
-        preferred_mails_from: 'spree@example.com'
-      )
+      Spree::Config[:mails_from] = "spree@example.com"
     end
     before { order5.cancel }
 
     it "only returns items from non-cancelled orders in the OC, placed by the user at the shop" do
       items = oc.items_bought_by_user(user, shop)
+      items.sort_by!(&:id)
```

Comment:
This seems completely unrelated to the task. A separate commit could have been good. And did you know about `match_array`?

Reply:
ok, moved this into a separate commit.
also used match_array instead, thanks, I didnt know it.